packages/cli/src/validators/schemas.ts

```diff
@@ -0,0 +1,341 @@
+                                    },
+                                    nullable: true,
+                                    required: ['type', 'const'],
+                                    additionalProperties: false,
+                                },
+                                whereable: {
+                                    type: 'object',
+                                    properties: {
+                                        type: {
+                                            type: 'string',
+                                        },
+                                        const: {
+                                            type: 'boolean',
+                                        },
+                                    },
+                                    nullable: true,
+                                    required: ['type', 'const'],
+                                    additionalProperties: false,
+                                },
+                                nullable: {
+                                    type: 'object',
+                                    properties: {
+                                        type: {
+                                            type: 'string',
+                                        },
+                                        const: {
+                                            type: 'boolean',
+                                        },
+                                    },
+                                    nullable: true,
+                                    required: ['type', 'const'],
+                                    additionalProperties: false,
+                                },
+                                isPrimaryKey: {
+                                    type: 'object',
+                                    properties: {
+                                        type: {
+                                            type: 'string',
+                                        },
+                                        const: {
+                                            type: 'boolean',
+                                        },
+                                    },
+                                    nullable: true,
+                                    required: ['type', 'const'],
+                                    additionalProperties: false,
+                                },
+                            },
+                            required: [],
+                            additionalProperties: false,
+                        },
+                    },
+                    required: [],
+                    additionalProperties: false,
+                },
+            },
+            nullable: true,
+            required: [],
+            additionalProperties: false,
+        },
+    },
+    required: [],
+    additionalProperties: false,
+};
+
+export const cliArgvSchema: JSONSchemaType<CliConfig> = {
```

Comment:
I'm not sure why this is needed 🤔 doesn't `yargs` provide type-safety already?

Reply:
It does, but because schemaDefOverrides is read from the config file, it's type is unknown to yargs, so we use that to give the values their correct type

Reply:
Ok fair, but that means we just need a schema for the schemaDefOverrides, not for all the other args, right?

Reply:
That is correct... I will adjust it now 👍🏾